vanilla/vanilla
# Add styles fixes to keystone and theme-boilerplate

Closes https://github.com/vanilla/support/issues/2181
Closes https://github.com/vanilla/support/issues/1617
Closes https://github.com/vanilla/support/issues/2069
Closes https://github.com/vanilla/vanilla/issues/9383
Closes: https://github.com/vanilla/support/issues/2238

**Steps to test:**
- Check each scenarios on the issues above
- Checkout this branch
- See the visual difference
